core/templates/dev/head/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -130,11 +138,20 @@ oppia.directive('questionsList', [
             }
             if (!ctrl.questionIsBeingUpdated) {
               EditableQuestionBackendApiService.createQuestion(
-                ctrl.skillId, ctrl.question.toBackendDict(true)
-              ).then(function() {
-                ctrl.fetchQuestionSummaries(ctrl.entityId, true);
-                ctrl.questionIsBeingSaved = false;
-                ctrl.currentPage = 0;
+                ctrl.skillIds[0], ctrl.question.toBackendDict(true)
```

Comment:
Instead can we create the question directly with all the skills linked?

Reply:
Done.